python/lsst/ts/observatory/control/utils/enums.py

```diff
@@ -30,17 +30,39 @@ class RotType(enum.IntEnum):
          to the North axis so rot_angle=0. means y-axis is aligned with North.
          Angle grows clock-wise.
 
+    SkyAuto: Same as sky position angle but it will verify that the requested
```

Comment:
I think this could be clearer without the term "strategy". For example:

"""Available types of rotation, i.e. the meaning of the rot_angle argument.

Sky: rotate with the sky. The rotator is positioned...
...
PhysicalSky: start at the specified physical angle (see Physical), then rotate with the sky.
```